packages/compass-aggregations/src/components/aggregation-side-panel/stage-wizard-use-cases/sort/sort.tsx

```diff
@@ -0,0 +1,182 @@
+import {
+  Select,
+  Option,
+  IconButton,
+  Icon,
+  Body,
+  spacing,
+  css,
+  ComboboxWithCustomOption,
+} from '@mongodb-js/compass-components';
+import React, { useEffect, useMemo, useState } from 'react';
+import { connect } from 'react-redux';
+import type { RootState } from '../../../../modules';
+
+const SORT_DIRECTION_OPTIONS = [
+  {
+    label: 'Ascending',
+    value: 'Asc',
+  },
+  {
+    label: 'Descending',
+    value: 'Desc',
+  },
+] as const;
+
+type SortDirection = typeof SORT_DIRECTION_OPTIONS[number]['value'];
+type SortFieldState = {
+  field: string;
+  direction: SortDirection;
+};
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[2],
+});
+
+const formGroupStyles = css({
+  display: 'flex',
+  justifyContent: 'flex-start',
+  alignItems: 'center',
+  gap: spacing[2],
+});
+
+const labelStyles = css({
+  minWidth: '140px',
+  textAlign: 'right',
+});
+
+const sortDirectionStyles = css({
+  width: '150px',
+});
+
+const mapSortFormDataToStageValue = (
+  formData: SortFieldState[]
+): Record<string, number> => {
+  return formData.reduce((acc, sort) => {
+    if (sort.field) {
+      acc[sort.field] = sort.direction === 'Asc' ? 1 : -1;
+    }
+    return acc;
+  }, {} as Record<string, number>);
```

Comment:
```suggestion
  return formData.reduce<Record<string, number>>((acc, sort) => {
    if (sort.field) {
      acc[sort.field] = sort.direction === 'Asc' ? 1 : -1;
    }
    return acc;
  }, {});
```
Very small thing, can be ignored.
For this we could tell reduce directly the return type. Though the type-check outcome will be similar I guess.